phonebook.py

```diff
@@ -2,7 +2,34 @@
 class Phonebook:
 
     def __init__(self):
-        pass
+        self.phonebook = {}
 
-    def add_contacts():
-        pass
+    def add_contact(self, name, contact):
+
+        if contact.isdigit() and (name != "" or name != " "):
+            self.phonebook[name] = contact
+            return {"message": "Contact successfully added!!"}
+
+        else:
+            raise ValueError
+
+    def view_contact(self, name):
+        view = self.phonebook.get(name, "Empty")
+        if view == "Empty":
+            return {"message": "The contact is missing!!"}
+
+        return self.phonebook[name]
```

Comment:
What do you think about looping through the dictionary and return a list of a set with a name, contact pair? This is because the phonebook can contain more than one record of contacts